Sources/Service/Environment/Environment+DotEnv.swift

```diff
@@ -8,30 +8,30 @@ import Core
 
 extension Environment {
     /// Loads the ".env" and the ".env.environment" files if they exist
-    public func loadDotEnv() {
+    public func loadEnvironment() {
         // Load the default one
-        self.loadDotEnvFile(".env")
+        self.load(envFile: ".env")
         
         // Load any environment specific ones next
-        self.loadDotEnvFile(".env.\(self.name)")
+        self.load(envFile: ".env.\(self.name)")
     }
     
     /// Loads the specific env file and sets the appropiate environment variables
-    public func loadDotEnvFile(_ filename: String) {
+    public func load(envFile filename: String) {
         // Create the file path
         let url = URL(fileURLWithPath: DirectoryConfig.detect().workDir).appendingPathComponent(filename)
         
         // Read the contents of the file
         guard let contents = try? String(contentsOf: url, encoding: .utf8) else { return }
         
         // Loop through all the key/values and set the environment variables
-        self.parseDotEnv(with: contents).forEach { (key, value) in
+        self.parse(with: contents).forEach { (key, value) in
             setenv(key, value, 1)
         }
     }
     
     /// Parse the contents of the .env file into a readable key/value
-    func parseDotEnv(with content: String) -> [String : String] {
+    func parse(with content: String) -> [String : String] {
```

Comment:
Now that you removed the `DotEnv` from the method name there is nothing that tells the caller what type of content it is wanting to parse.

Maybe would should change it to `func parse(withDotEnv content: String)`?

Reply:
`func parse(dotEnv content: String)`? not a big fan of `with` in method signatures, also don't think they much used in vapor?

Reply:
This is an internal function, so the user shouldn't ever see this function